### ISSUE TITLE
Check functionality of JSX && operator v1 and v2

### ISSUE THREAD

**Issue by stephanschroeter** [porsche-design-system/porsche-design-system#933]
Scope
---
After upgrading to the latest Stencil version the `&&` operator does not work correctly (on conditions inside of attributes e.g. aria-selected={this.optionSelected === key && 'true' }) anymore in JSX template context of our components. As an alternative we can use `condition ? 1 : 2`. We need to check if we use somewhere the `&&` operator in component's JSX template and if so this means there's no test coverage for it which also needs to be fixed by adding a test.

Notes
---
Short time ago we've updated the project from Stencil v1 to v2,… maybe it's worth checking if the `&&` operator works again.
The `&&`-operator also affects the change detection which might not work at all, therefor we need an E2E test.

- See https://github.com/porscheui/porsche-design-system/pull/919/files

Findings
---

Error is caused due to render cache of stencil. && operator works but we have to re-trigger the rendering via a uniq key attribute

Acceptance Criteria
---
- [x] `&&` operator in JSX templates is replaced with `condition ? 1 : 2`
- [x] Conditional operators (logic) are fully tested by test automation 

Subtasks
---
- [x] Check every component for && operater in attributes
- [x] Add test for aria attributes Button and Button Pure
- [x] Add test for aria modal?
- [x] check test aria for checkBoxsWrapper
- [x] check test aria for radio
- [x] check test aria for select
- [x] check test aria for text-field
- [x] check test aria for text-area
- [x] check test aria for select
- [x] add test aria for link
- [x] add test aria for link-pure
- [x] add test aria for link-social
- [x] add test aria for pagination
- [x] add test aria for tabs-bar
